### PR TITLE
Added menu navigation

### PR DESCRIPTION
**Pull request by AdrianCSM**
I have added the mobile navigation menu with Lekan Jimoh.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,8 +20,12 @@
               <img src="images/MyLogo.svg" alt="Logo" />
             </a>
             <nav class="dropdown">
-              <img src="images/NavMenu.svg" alt="Nav Menu" class="nav_menu" />
-              <ul class="mobile_nav">
+                <img src="images/NavMenu.svg" alt="Nav Menu" class="nav_menu" />
+              <ul class="mobileNav">
```
Comment (elyor-doniyorov) [inline index.html:24]: - [ ] Please make sure the mobile menu follows the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, tests, margins). Currently, your `design color` for text is not` white color` like it in the Figma design. Take a look at the Figma style before making necessary changes to styling 👇 
- Figma design color 👇 
Screenshot (208)
- Yours 👇 
Screenshot (209)

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @AdrianCSM, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- I do recommend adding GitHub repo issues page link and license to the Readme file;

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (AdrianCSM) [reply index.html:24]: Done. I've changed the colors. I really forgot about them, thank you very much for pointing this out.

**Review by FrederickMih — APPROVED** [review]
# APPROVED

Hi @AdrianCSM , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**